src/main/java/duke/task/Events.java

```diff
@@ -0,0 +1,40 @@
+package duke.task;
+
+/**
+ * Encapsulates a task to be done at a specified time.
+ */
+public class Events extends Task {
```

Comment:
You might want to rename `Events` to `Event` since the class represents an `Event` task